tools/mmp/Application.cs

```diff
@@ -13,8 +13,6 @@ public partial class Application
 
 		internal void Initialize ()
 		{
-			if (DeploymentTarget == null) 
-				DeploymentTarget = new Version (10, 7);
```

Comment:
from what I can see the new code would return @OSX_SDK_VERSION@ which is 10,13

Reply:
it might be best to leave the initialization (if null) to the tool (not shared)
and only move the validation to the common code

Reply:
Looks like you're right.  iOS and mac currently handle this differently (iOS sets to latest version, macOS sets to 10.7).  I'll move these back to the separate spots.